releases/sentry.yaml

```diff
@@ -0,0 +1,288 @@
+          ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes)
+          livenessProbe:
+            enabled: true
+            initialDelaySeconds: 30
+            periodSeconds: 5
+            timeoutSeconds: 2
+            failureThreshold: 3
+            successThreshold: 1
+          readinessProbe:
+            enabled: true
+            initialDelaySeconds: 30
+            periodSeconds: 5
+            timeoutSeconds: 2
+            failureThreshold: 3
+            successThreshold: 1
+
+          ## Metrics exporter resource requests and limits
+          ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+          resources:
+            limits:
+              cpu: '{{ env "SENTRY_METRICS_RESOURCES_LIMIT_CPU" | default "300m" }}'
+              memory: '{{ env "SENTRY_METRICS_RESOURCES_LIMIT_MEMORY" | default "256Mi" }}'
+            requests:
+              cpu: '{{ env "SENTRY_METRICS_RESOURCES_REQUEST_CPU" | default "100m" }}'
+              memory: '{{ env "SENTRY_METRICS_RESOURCES_REQUEST_MEMORY" | default "128Mi" }}'
+
+          nodeSelector: {}
+          tolerations: []
+          affinity: {}
+          # schedulerName:
+          # Optional extra labels for pod, i.e. redis-client: "true"
+          # podLabels: []
+          service:
+            type: ClusterIP
+            labels: {}
+
+          image:
+            repository: prom/statsd-exporter
+            tag: '{{ env "SENTRY_METRICS_IMAGE_TAG" | default "v0.10.5" }}'
+            pullPolicy: IfNotPresent
+
+          # Enable this if you're using https://github.com/coreos/prometheus-operator
+          serviceMonitor:
+            enabled: {{ env "SENTRY_METRICS_SERVICE_MONITOR_ENABLED" | default "false" }}
+            ## Specify a namespace if needed
+            # namespace: monitoring
+            # fallback to the prometheus default unless specified
+            # interval: 10s
+            ## Defaults to what's used if you follow CoreOS [Prometheus Install Instructions](https://github.com/helm/charts/tree/master/stable/prometheus-operator#tldr)
+            ## [Prometheus Selector Label](https://github.com/helm/charts/tree/master/stable/prometheus-operator#prometheus-operator-1)
+            ## [Kube Prometheus Selector Label](https://github.com/helm/charts/tree/master/stable/prometheus-operator#exporters)
+            selector:
+              prometheus: kube-prometheus
+
+        # Provide affinity for hooks if needed
+        hooks:
+          affinity: {}
+          dbInit:
+            resources:
+              # We setup 3000Mi for the memory limit because of a Sentry instance need at least 3Gb RAM to perform a migration process
+              # reference: https://github.com/helm/charts/issues/15296
+              limits:
+                memory: '{{ env "SENTRY_HOOKS_RESOURCES_LIMIT_MEMORY" | default "3200Mi" }}'
```

Comment:
Yikes! This needs to be called out, because that is practically a full t3.medium (4 GB total memory, but there are a lot of things that run on every node).